### DISCUSSION TITLE
Clicking on selected day again removes the format

### DISCUSSION THREAD

**Discussion by ArunChockalingam** [gpbl/react-day-picker#1117] [Support]
Code Sandbox: https://codesandbox.io/s/5v1xpq01on

Steps to reproduce:

1. choose a day (format will be MM/DD/YYYY)
2. choose the same day again (format will change to YYYY-MM-DD)

If you choose a different day, then it will go back to MM/DD/YYYY.

Can you please help with this issue or suggest other alternatives. Thanks!

**Comment by wenzoto** [comment]
I think will help  https://codesandbox.io/s/rrqz6nkljn

**Comment by ArunChockalingam** [comment]
My use case is i use the DayPickerInput for both the day view and the week view. I have implemented the selecting the entire week as part of this as well. So when the calendar is in day mode, i would like the value to be in the format of mm/dd/yyyy and when it is in week view , the value of the input has to be in the format of mm/dd/yyyy-mm/dd/yyyy.

 thats the reason why i am formatting the value directly similar to the code sandbox i posted above and thats when i ran into this issue. 

@gpbl Please let me know if i am approaching this incorrectly.

**Comment by Flaeron** [comment]
I have the same trouble :(
